fe/fe-core/src/main/java/com/starrocks/catalog/ListPartitionInfo.java

```diff
@@ -433,4 +441,18 @@ public void createAutomaticShadowPartition(long partitionId, String replicateNum
         idToInMemory.put(partitionId, false);
         idToStorageCacheInfo.put(partitionId, new DataCacheInfo(true, false));
     }
+
+    public static int compareByValue(List<List<String>> left, List<List<String>> right) {
```

Comment:
maybe this function has performance problem.

Reply:
why must use string? is there int partition value?

Reply:
Because List partition serialization uses String, Expr is not easy to serialize. So unified into this structure.